alvr/xtask/src/main.rs

```diff
@@ -291,7 +309,7 @@ pub fn build_client(is_release: bool, is_nightly: bool, for_oculus_go: bool) {
 
     let client_dir = workspace_dir().join("alvr/client/android");
     let command_name = if cfg!(not(windows)) {
-        "gradlew"
+        "./gradlew"
```

Comment:
Thank you (:

Reply:
Without the leading ./ it wouldn't work on macos.

Reply:
And on Linux I had to add that folder to the path every time. :P